src/test/javascript/spec/component/programming-exercise/programming-exercise-update.component.spec.ts

```diff
@@ -73,48 +63,40 @@ describe('ProgrammingExercise Management Update Component', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, MockModule(NgxDatatableModule), MockModule(NgbModule), MockModule(OwlDateTimeModule)],
+            imports: [ArtemisTestModule, MockModule(NgxDatatableModule)],
             declarations: [
                 ProgrammingExerciseUpdateComponent,
-                MockComponent(AlertComponent),
-                MockComponent(AlertErrorComponent),
-                MockComponent(HelpIconComponent),
+                NgForm,
                 NgModel,
                 CheckboxControlValueAccessor,
                 DefaultValueAccessor,
                 SelectControlValueAccessor,
                 NumberValueAccessor,
```

Comment:
Does this need to be imported raw? If not, please mock it. Otherwise, please specify why it does.

Reply:
Unfortunately yes. Without the real form accessors the SCA tests fail because the values from the inputs like the `maxStaticCodeAnalysisPenalty` isn't set properly anymore.

Reply:
Could you please check if there is relevant functionality? If so, we should consider manual mocks, if not, I'd ask you to add a short inline comment explaining why we decided to import this raw.